test/pytest/conftest.py

```diff
@@ -44,7 +44,8 @@ def pytest_ignore_collect(path: object) -> bool:
 
 
 @pytest.fixture
-def event_loop_policy() -> asyncio.AbstractEventLoopPolicy:
+# AbstractEventLoopPolicy is deprecated in 3.14 and will be removed in 3.16
+def event_loop_policy() -> asyncio.AbstractEventLoopPolicy:  # type: ignore[name-defined]
```

Comment:
Throw some quotes on there to help prevent runtime problems in the future...

Although hopefully by then they'll have the lazy-evaluation in place...